easybuild/easyconfigs/s/scikit-learn/scikit-learn-0.20.3-foss-2019a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'scikit-learn'
+version = '0.20.3'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
+description = """Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
+building upon numpy, scipy, and matplotlib. As a machine-learning module,
+it provides versatile tools for data mining and analysis in any field of science and engineering.
+It strives to be simple and efficient, accessible to everybody, and reusable in various contexts."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['c503802a81de18b8b4d40d069f5e363795ee44b1605f38bc104160ca3bfe2c41']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [('SciPy-bundle', '2019.03')]
+
```

Comment:
The matplotlib dependency is missing.
And previous scikit-learn easyconfigs have been using matplotlib 3 for python 3 builds and matplotlib 2 for python 2 builds.

Not quite sure how important matplotlib is for the building of scikit-learn.

Reply:
`matplotlib` isn't really a dependency of `scikit-learn`, but it is very useful when combined with it

Not listing `matplotlib` as a dependency makes sense because of that, but it also enables us to install `scikit-learn` for both Python 3.7.2 & 2.7.15 (since latest `matplotlib` 3.x requires Python 2).

Reply:
indeed... hadn't looked at what scikit-learn really needs.